src/Sulu/Bundle/AdminBundle/Admin/View/FormOverlayListViewBuilder.php

```diff
@@ -186,6 +186,13 @@ public function setOverlaySize(string $overlaySize): FormOverlayListViewBuilderI
         return $this;
     }
 
+    public function addFormMetadata(array $formMetadata): FormOverlayListViewBuilderInterface
```

Comment:
@sulu/core-team WDYT about this naming? Currently we have only `addRouterAttributesToListMetadata` being similar. There is also `setRequestParameters` on the `FormViewBuilder`, which is similar in regard that the options being passed are hardcoded.

Should we leave the short version `addFormMetadata` here or go for something like `addParametersToFormMetadata`?

Reply:
this adds options that are appended to the metadata request right? that would be similar to `setRequestParameters`, but for another request. what do you think about `setMetadataRequestParameters`?